src/AccessibilityInsights.SetupLibrary/VersionSwitcherWrapper.cs

```diff
@@ -118,6 +118,30 @@ private static void RecursiveTreeCopy(string source, string dest, List<FileStrea
                 string destFile = Path.Combine(dest, fileInfo.Name);
                 fileInfo.CopyTo(destFile, true);
                 fileLocks.Add(File.OpenRead(destFile));
+                EnsureFilesAreIdentical(file, destFile);
+            }
+        }
+
+        private static void EnsureFilesAreIdentical(string file1, string file2)
+        {
+            byte[] file1Bytes = File.ReadAllBytes(file1);
+            byte[] file2Bytes = File.ReadAllBytes(file2);
+
+            bool matches = file1Bytes.Length == file2Bytes.Length;
```

Comment:
You could use
```suggestion
            bool matches = file1Bytes.SequenceEqual(file2Bytes);
```
here and then skip the manual iteration/equality checking below. Presumably doesn't make much of a difference

Reply:
Always happy to have less code 😄